Forta-Bots-Challenge-2/src/agent.spec.ts

```diff
@@ -0,0 +1,216 @@
+import { HandleTransaction, Initialize } from "forta-agent";
+import { provideHandleTransaction, provideInitialize } from "./agent";
+import { UNISWAP_POOL_ABI, ERC20_ABI, SWAP_EVENT } from "./constants";
```

Comment:
Same comment as the fetcher test suite. (i.e. recreating the constants in the test file)